### DISCUSSION TITLE
Wifi - & BOOTSLE - is it possible to load new code over Wifi?

### DISCUSSION THREAD

**Discussion by FlydGitHub** [micropython/micropython#11279] [RP2040 / Pico]
Is it possible to load new code over Wifi?

**Comment by Josverl** [comment]
not sure what you mean by `BOOTSLE`
but I understand it as : Is it possible to do an OTA on a Pico-W 

The Pico-W can :  See https://github.com/usedbytes/picowota
but AFAIK this is not part of the Micropython port for the rp2/pico-w.

**Comment by FlydGitHub** [comment]
Thanks that looks like what I need.

Is the a Micropython version of the link you sent?

**Comment by FlydGitHub** [comment]
Is it possible to do a  BOOTSLE over wifi too?

**Reply by karfas** [reply]
What exactly do you mean with "BOOTSLE" ?

**Comment by FlydGitHub** [comment]
Load a new firmware image, as in https://micropython.org/download/rp2-pico-w/

**Reply by karfas** [reply]
The button is called "**BOOTSEL**". Even when it looks like I'm nitpicking here - python or any other programming language is far more pickier than me.
